src/pkg/packager/creator/creator_test.go

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2021-Present The Zarf Authors
+
+// Package creator contains functions for creating Zarf packages.
+package creator
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/defenseunicorns/zarf/src/pkg/layout"
+	"github.com/defenseunicorns/zarf/src/pkg/packager/lint"
+	"github.com/defenseunicorns/zarf/src/types"
+	"github.com/stretchr/testify/require"
+)
+
+type mockSchemaLoader struct {
+	b []byte
+}
+
+func (m *mockSchemaLoader) ReadFile(_ string) ([]byte, error) {
+	return m.b, nil
+}
+
+func TestLoadPackageDefinition(t *testing.T) {
+	// TODO once creator is refactored to not expect to be in the same directory as the zarf.yaml file
+	// this test can be re-parallelized
+	tests := []struct {
+		name        string
+		testDir     string
+		expectedErr string
+		creator     Creator
+	}{
+		{
+			name:        "valid package definition",
+			testDir:     "valid",
+			expectedErr: "",
+			creator:     NewPackageCreator(types.ZarfCreateOptions{}, ""),
+		},
+		{
+			name:        "invalid package definition",
+			testDir:     "invalid",
+			expectedErr: "errors during lint",
+			creator:     NewPackageCreator(types.ZarfCreateOptions{}, ""),
+		},
+		{
+			name:        "valid package definition",
+			testDir:     "valid",
+			expectedErr: "",
+			creator:     NewSkeletonCreator(types.ZarfCreateOptions{}, types.ZarfPublishOptions{}),
+		},
+		{
+			name:        "invalid package definition",
+			testDir:     "invalid",
+			expectedErr: "errors during lint",
+			creator:     NewSkeletonCreator(types.ZarfCreateOptions{}, types.ZarfPublishOptions{}),
+		},
+	}
+	b, err := os.ReadFile("../../../../zarf.schema.json")
```

Comment:
I would suggest a changeDir, but then we would have to move back. 🤔  We could copy? 
This isn't a big deal, but it does look ugly, and it we move this file it'll cause problems. (low priority nit)

Reply:
Yeah I guess no matter what we still have to reference the file, kinda a pick your poison situation. This seems simplest.